lib/agent.js

```diff
@@ -6,6 +6,10 @@
  * See the included LICENSE file for more details.
  */
 
+// Polyfill for `String.prototype.normalize`.
+// To be removed when Node 0.x is no longer supported.
+require('unorm')
```

Comment:
I did not find a package only exporting the normalization function.

Could use `const nfc = require('unorm').nfc` instead but `String.prototype` will still be extended.